app/navigation/AuthorisedOfficialsNavigator.scala

```diff
@@ -60,6 +60,12 @@ class AuthorisedOfficialsNavigator @Inject()(implicit frontendAppConfig: Fronten
       case _ =>  routes.SessionExpiredController.onPageLoad()
     }
 
+    case AuthorisedOfficialPreviousAddressPage(index) => userAnswers:UserAnswers  => userAnswers.get(AuthorisedOfficialPreviousAddressPage(index)) match {
+      case Some(true) => routes.DeadEndController.onPageLoad() // TODO redirect to next page once created
```

Comment:
can we combine this please Some(_)

Reply:
Would it make sense to keep the true/false logic for the later pages in the flow?